src/remote.go

```diff
@@ -66,7 +66,8 @@ const (
 var (
 	ErrPathNotExists                  = errors.New("remote path doesn't exist")
 	ErrNetLookup                      = errors.New("net lookup failed")
-	ErrClashesDetected                = fmt.Errorf("clashes detected. use `%s` to override this behavior", CLIOptionIgnoreNameClashes)
+	ErrClashesDetected                = fmt.Errorf("Clashes detected. Use `%s` to override this behavior or `%s` to try fixing this",
```

Comment:
I don't agree with that (I think each error should be a full sentence) but sure we can. But having lowercase letter after dot looks strange. Should I leave it uppercase or change it back to lowercase?